frontend/automation/cypress/integration/transaction-approval.feature

```diff
@@ -1,10 +1,8 @@
 Feature: Transaction approval
 
-  Background:
+  Scenario: Transaction approved
```

Comment:
Removing the background step to the scenario would mean the tests can't be run independently. Would be nice if we could avoid this but also at some point we'd want to take out BDD from this project so not blocking.